sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -1041,9 +1041,9 @@ describe("Atom management - Authentication", function(): void {
     input: undefined,
     output: {
       authorizationRules: undefined,
-      autoDeleteOnIdle: "P10675199DT2H48M5.4775807S",
-      defaultMessageTtl: "P10675199DT2H48M5.4775807S",
-      duplicateDetectionHistoryTimeWindow: "PT10M",
+      autoDeleteOnIdleInSeconds: 10675199 * 24 * 60 * 60 + 2 * 60 * 60 + 48 * 60 + 5.4775807, //"P10675199DT2H48M5.4775807S",
```

Comment:
I'm having a hard time telling...do the atomManagement tests actually test that a response with the string duration is deserialized into the correct number?

Reply:
Not exactly, I have added separate unit tests for that - "string duration is deserialized into the correct number".

For this test, we do verify the response by matching it with this predefined output.
The response would contain the iso-8601 duration converted into seconds. 
If the number from the response matches with the number here in the predefined output, the test would succeed and I would assume the string duration is deserialized into the correct number.